src/groups/mqb/mqbi/mqbi_storage.h

```diff
@@ -408,6 +408,10 @@ class Storage {
                           const bsls::Types::Int64 messageTtl,
                           const int                maxDeliveryAttempts) = 0;
 
+    /// Set the consistency level associated to this storage to the specified
+    /// `value`.
+    virtual bool setConsistency(const mqbconfm::Consistency& value) = 0;
```

Comment:
I decided to introduce a separate setter instead of extending `configure`, because sometimes we call `configure` without setting new consistency.